production/docker/docker-compose-ha-memberlist.yaml

```diff
@@ -5,15 +5,30 @@ networks:
 
 services:
 
+  # Since the Loki containers are running as user 10001 and the mounted data volume is owned by root,
+  # Loki would not have permissions to create the directories.
+  # Therefore the init container changes permissions of the mounted directory.
+  init:
+    image: grafana/loki:2.4.1
+    user: root
+    entrypoint:
+      - "chown"
+      - "10001:10001"
```

Comment:
Why is this required?

Reply:
This is required because the mounted directories have the permissions of the host user.